components/button/buttonPage.js

```diff
@@ -72,4 +74,15 @@ function setAllCodeContainerDisplayForRowNone(eleRowIdx) {
     })
 }
 
-setDefault();
\ No newline at end of file
+setDefault();
+
+// Add buttin listener for hamburger functionality
+const moblFltMenu = document.querySelector(".mobl-flt-menu");
```

Comment:
Suggestion : maybe a common js file with the below said function which can be called when required rather than repeating in same file.

Reply:
Will be refactoring it in the fututure.